logservice/schemastore/persist_storage_ddl_handlers.go

```diff
@@ -1623,46 +1624,31 @@ func buildDDLEventForRenameTable(rawEvent *PersistedDDLEvent, tableFilter filter
 				InfluenceType: commonEvent.InfluenceTypeNormal,
 				TableIDs:      allPhysicalIDsAndDDLSpanID,
 			}
-			if !ignoreCurrentTable {
```

Comment:
This check cannot be removed. The following fields is used to update meta info for the changefeed and check which dispatcher need be removed.
BTW. It is not suggested to put unrelated changes in the same big pr.